workers/unity/Assets/Gdk/Core/GameObjectRepresentation/GameObjectDispatcherCallbacksSystem.cs

```diff
@@ -0,0 +1,70 @@
+using Unity.Entities;
+
+namespace Improbable.Gdk.Core
+{
+    /// <summary>
+    ///     Gathers incoming dispatcher ops and invokes callbacks on relevant GameObjects.
+    /// </summary>
+    [UpdateInGroup(typeof(SpatialOSUpdateGroup))]
```

Comment:
This should probably be done earlier. I'd suggest after the initialisation group and before the update group.

Reply:
Added a new update group between `EntityInitialisationGroup` and `SpatialOSUpdateGroup`.